model/fv_regional_bc.F90

```diff
@@ -6632,25 +6632,38 @@ end subroutine exch_uv
 !+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 !---------------------------------------------------------------------
 
-  subroutine get_data_source(source,regional)
+  subroutine get_data_source(data_source_fv3gfs,regional)
 !
 ! This routine extracts the data source information if it is present in the datafile.
 !
-      character (len = 80) :: source
-      integer              :: ncids,sourceLength
+      character (len=80) :: source      
       logical :: lstatus,regional
+      logical :: data_source_fv3gfs
 !
 ! Use the fms call here so we can actually get the return code value.
+! The term 'source' is specified by 'chgres_cube'
 !
       if (regional) then
        lstatus = get_global_att_value('INPUT/gfs_data.nc',"source", source)
       else
        lstatus = get_global_att_value('INPUT/gfs_data.tile1.nc',"source", source)
       endif
+      if (mpp_pe()==0) write(*,*) 'INPUT gfs_data source string=',source
```

Comment:
Consider move this line after the if lstatus statment (6652-6655)

Reply:
Completed